NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -502,9 +502,11 @@ let SkipTimeInterval: Double = 15
     }
 
     func middleTextFor(chapter: ChapterLocation) -> String {
-        let middleTextFormat = NSLocalizedString("Part %@ of %d", bundle: Bundle.audiobookToolkit()!, value: "Part %@ of %d", comment: "Current chapter and the amount of chapters left in the book")
+        let defaultTitleFormat = NSLocalizedString("Chapter %@", bundle: Bundle.audiobookToolkit()!, value: "Chapter %@", comment: "Default chapter title")
+        let middleTextFormat = NSLocalizedString("%@ (file %@ of %d)", bundle: Bundle.audiobookToolkit()!, value: "%@ (file %@ of %d)", comment: "Current chapter and the amount of chapters left in the book")
```

Comment:
this string should also be added to the localized strings file.  Similar question about the `value` param (here and in all the other changes) if we can omit it it would be a little easier to read.